backend/workitem.go

```diff
@@ -11,7 +11,8 @@ import (
 var ErrNoWorkItems = errors.New("no work items were found")
 
 type WorkItem interface {
-	Description() string
+	fmt.Stringer
+	IsWorkItem() bool
```

Comment:
I assume this must be a Go thing, but given nobody actually consumes this and given that all implementations return `true`, am I correct to assume that this was added just to distinguish work items from other random types?

Reply:
Yes that's precisely it. It's just a Go pattern, for example gRPC does the same